icons.yml

```diff
@@ -109,9 +109,21 @@ mappings:
   - subclass: community_centre
     iconName: town-hall-11
     color: '#8A9CF6'
-  - subclass: place_of_worship
+  - class: place_of_worship
```

Comment:
The use of `class` without `subclass` is not implemented yet.
For now, it matches empty subclass (with any class)

Reply:
`- class: place_of_worship` without subclass should generate the following snippet in the style:
```
          [
            "==",
            [
              "get",
              "class"
            ],
            "place_of_worship"
          ],
          "place-of-worship-11",
```
But we should add this declaration after the more specific one so it can be used as a fallback.

Reply:
https://github.com/QwantResearch/map-style-builder/pull/15